src/sm/sm_main/sm_main_exec.js

```diff
@@ -280,8 +290,13 @@ module.exports = async function execute(pols, input, rom, config = {}) {
         }
 
         if (l.inSTEP) {
-            op0 = Fr.add(op0, Fr.mul( Fr.e(l.inSTEP), Fr.e(i)));
-            pols.inSTEP[i] = Fr.e(l.inSTEP);
+            if (skipCounters) {
+                op0 = 1n;
```

Comment:
I known that skipCounters was a exceptional situation, but for me was better: 
op0 = Fr.add(Fr.one, Fr.mul( Fr.e(l.inSTEP), Fr.e(i)));

Reply:
I would maintain `Fr.one` just to increase the step